source/javascripts/controllers/_MainController.js.erb

```diff
@@ -742,5 +745,9 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 			viewModel.appNameWithPlaceholder = function() {
 				return appService.appDetails.title || 'Unknown app';
```

Comment:
Unrelated: do we really have apps with falsy titles?

Reply:
I'm not sure, might be for defensive or backward compatibility reasons?